wiki/Contests/Official_support/en.md

```diff
@@ -72,6 +73,19 @@ If approved, you will need to submit the following:
 
 In exceptionally rare cases, a contest may be granted the privilege of awarding badges to its top three finalists. A contest that has received this distinction is recognised for attaining the highest level of production quality and organisation possible — this is given exclusively at the discretion of the osu! team. Hosts will be contacted directly if applicable.
 
+### News posts
```

Comment:
any contest which has news posts would use the contest listing too, right?

Reply:
maybe as a note

```suggestion
### News posts

*Note: News coverage of a contest typically also implies that it will have an entry on the [contest listing](https://osu.ppy.sh/community/contests).*
```

... i'm not entirely certain if this will hold true in all cases but i can't think of any counterexamples

Reply:
Currently, using the Contest listing in the MPG site is only a must if they're going for contest points. Contests can still receive official support without using it, and therefore be eligible for a news post.